src/layouts/index.jsx

```diff
@@ -144,6 +145,35 @@ const Layout = ({ children, location }) => {
         </div>
         <Sidebar location={location} />
       </div>
+      <CookieConsent
+        className="mama"

```

Comment:
Is “mama” an intended class name? (Quick DOM search doesn’t show it—is it even part of the output?)